nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/AbstractAzureBlobProcessor_v12.java

```diff
@@ -83,45 +87,38 @@ public abstract class AbstractAzureBlobProcessor_v12 extends AbstractProcessor {
             .description("Unsuccessful operations will be transferred to the failure relationship.")
             .build();
 
+    protected static final List<PropertyDescriptor> PROPERTIES = Collections.unmodifiableList(Arrays.asList(
+            STORAGE_CREDENTIALS_SERVICE,
+            AzureStorageUtils.PROXY_CONFIGURATION_SERVICE
+    ));
```

Comment:
The abstract processor did not contain common properties intentionally. I strongly recommend to use simple property lists in processor classes instead. This way each processor can define its properties in the right order and without cumbersome code (adding/deleting properties).